src/common.h

```diff
@@ -0,0 +1,8 @@
+#define REDBOLD "\033[1;31m"
```

Comment:
Does this work in CMD? I think only powrshell supports that nice formatting. Actually, I was forced to remove it from hfl-dump due to many reports about this characters in cmd. Can you remove formatting for now, and add it later as separate PR?

Reply:
> Does this work in CMD? I think only powrshell supports that nice formatting. Actually, I was forced to remove it from hfl-dump due to many reports about this characters in cmd. Can you remove formatting for now, and add it later as separate PR?

or disable on windows. althoug, WSL and windows terminal do support it